shared/build/script/docker.go

```diff
@@ -37,3 +44,31 @@ func DockerHostname(d *Docker) string {
 	}
 	return *d.Hostname
 }
+
+// DockerVolumes returns empty []string
+// when Docker.Volumes is empty.
+func DockerVolumes(d *Docker) []string {
```

Comment:
we can remove this method and just access d.Volumes directly. This is because `d` will never be nil, and `d.Volumes` will never be nil, and if `len(d.Volumes) == 0` it is the same as returning an empty string slice :)